pom.xml

```diff
@@ -13,7 +13,7 @@
 
     <groupId>io.github.swagger2markup</groupId>
     <artifactId>swagger2markup-maven-plugin</artifactId>
-    <version>1.3.4</version>
+    <version>1.3.4-muddysteel-SNAPSHOT</version>
```

Comment:
Please revert the version to 1.3.5-SNAPSHOT

Reply:
done / pushed